wolfSSL/src/chapter02.md

```diff
@@ -1154,6 +2084,11 @@ Enable Single-Precision (SP) RISCV32 assembly speedup macros. Only applies if `W
 
 Enable Single-Precision (SP) S390X assembly speedup macros. Only applies if `WOLFSSL_SP_MATH_ALL` is defined. See `sp_int.c`.
 
+#### SP_INT_BITS
+
+Single-Precision (SP) assembly speedup macro. Math configuration that can specify size.
```

Comment:
`SP_INT_BITS` doesn't care about assembly optimization-ness.  It's just the "number of bits to have in an `sp_int`", which determines the largest bignum that the library can handle (see `wolfssl/wolfcrypt/sp_int.h`).